### DISCUSSION TITLE
Confuse while learning

### DISCUSSION THREAD

**Discussion by Nathan12212** [Cyfrin/foundry-full-course-cu#1584] [Q&A]
What should  i learn from basic to expert, to be bug hunter in web 3?

**Comment by EngrPips** [comment]
Hello @Nathan12212 , i would say just start with the first course on cyfrin updraft `blockchain basics` and up to the last course currently `Assembly and Formal Verification`. If you take those courses and understand everything in the courses then you are an expert in web3 and should be easy to get started with bug hunting.
